src/main/kotlin/no/nav/familie/ks/sak/api/dto/VilkårsvurderingDto.kt

```diff
@@ -39,3 +40,9 @@ data class AnnenVurderingDto(
     val type: AnnenVurderingType,
     val begrunnelse: String?
 )
+
+data class VedtakBegrunnelseTilknyttetVilkårDto(
```

Comment:
Dto-ene som brukes for respons, kan kalle som ResponsDto, så her VedtakBegrunnelseTilknyttetVilkårResponsDto. Det følgte vi i ks-sak på andre steder